WebApplication1/UserTaskList.aspx.cs

```diff
@@ -140,6 +145,42 @@ private int resolvePriorityRepeat(Label label)
                 return 2;
         }
 
+
+        private void progressbar(object sender, EventArgs e)
```

Comment:
it needs back and forth with data base, better to keep it shows as a date without progress bar!